app/views/organizations/index.html.erb

```diff
@@ -1,41 +1,51 @@
-<div>
-  <div id="map" style='height: 280px;'></div>
-</div>
+<div class = "header-container" >
+  <div class = "video-container">
+    <video preload="true" autoplay loop muted>
+      <source src="<%= video_path('skyline_clip.mp4') %>" type="video/mp4">
+      </video>
+    </div>
+    <h2>  VANCOUVER TECH HUB </h2>
+    <p> Discover exciting companies, find local meet-ups and host events </p>
+  </div>
+
+  <div>
+    <div id="map" style='height: 280px;'></div>
+  </div>
 
-<script type="text/javascript">
+  <script type="text/javascript">
   handler = Gmaps.build('Google');
   handler.buildMap({ provider: { scrollwheel: false, zoomControl: true }, internal: {id: 'map'}}, function(){
     markers = handler.addMarkers(<%= raw organizations_json(@organizations).to_json %>);
     handler.bounds.extendWith(markers);
     handler.fitMapToBounds();
     //handler.getMap().setZoom(17);
   });
-</script>
-<br>
-<input id="organization-search" type="text" class="form-control" placeholder="Search" name="search">
-<br>
+  </script>
+  <br>
+  <input id="organization-search" type="text" class="form-control" placeholder="Search" name="search">
+  <br>
   <div class="container-fluid text-center">
     <div class="row">
       <% @organizations.each do |organization| %>
       <a href="<%= organization_path(organization) %>">
-      <div class="org-container col-lg-3 col-md-4 col-sm-6 col-xs-12" data-id="<%=organization.id%>" >
-        <img src="<%= organization.logo%>" id="org-logo"/>
-        <h3><%= link_to organization.name, organization_path(organization), class: "link-to-org" %></h3>
-        <p><%= organization.employee_count %> Employees</p>
-      </div>
+        <div class="org-container col-lg-3 col-md-4 col-sm-6 col-xs-12" data-id="<%=organization.id%>" >
+          <img src="<%= organization.logo%>" id="org-logo"/>
+          <h3><%= link_to organization.name, organization_path(organization), class: "link-to-org" %></h3>
+          <p><%= organization.employee_count %> Employees</p>
+        </div>
       </a>
       <% end %>
     </div>
   </div>
-<div class="container-fluid text-center">
-  <%= paginate @organizations %>
-</div>
-<div class="container-fluid text-center">
-  <%= page_entries_info @organizations %>
-</div>
+  <div class="container-fluid text-center">
+    <%= paginate @organizations %>
+  </div>
+  <div class="container-fluid text-center">
+    <%= page_entries_info @organizations %>
+  </div>
 
 
   <%# if o.image.present? %>
-    <%#= image_tag o.image.url(:thumb) %>
+  <%#= image_tag o.image.url(:thumb) %>
   <%# end %>
   <%#= link_to o.name, organization_path(o) %>
```

Comment:
You should not commit comments. Lines 46 to 51 are unnecessary.
![](http://i.imgur.com/zatftpM.gif)